class UserException(Exception):
    def __init__(self, msg: str, verbosity: int | None = None): ...
